distsql/select_result.go

```diff
@@ -92,6 +97,13 @@ func (r *selectResult) fetchResp(ctx context.Context) error {
 		}
 		if resultSubset == nil {
 			r.selectResp = nil
+			if !r.durationReported {
```

Comment:
Why need this check? Can't Observe the time immediately？

Reply:
Nope, the original metric is used for tracking the total running time of data fetching, therefore we have to keep the behavior the same with the original metric.